### DISCUSSION TITLE
Gamecube HW1 and reset

### DISCUSSION THREAD

**Discussion by Colrut** [darthcloud/BlueRetro#789] [General]
Good afternoon.  I have successfully installed a blueretro HW1 spec on gamecube.  I have discovered that I can use the existing reset switch to control the boot button on the esp32 board and control the function (pairing, reset, etc).  I initially did this by cutting one of the pins on the gamecube switch (top outside) so that the pins carrying power into the switch (there are three pins that have continuity with each other when switch is open) are maintained but the one that returns the switch status to the cube is cut.  by wiring this cut pin from the switch to the boot button on the eps32 board I can use the switch to control it.

as an experiment, I reconnected the return pin of the gamecube reset switch back to the controller board.  it still works as desired and will reset the gamecube and 'push' the boot switch at the same time.

I am trying to find a way to be able to reset the gamecube without triggering the boot button on the eps32.  it looks as if the HW2 spec will allow this.  Is it as simple as installing the HW2 firmware?  Will this allow a brief press of reset button (< 3 seconds) to reset the cube without disconnecting controllers?  Will HW2 cause issues on a board wired for HW1?  I understand I will not get the port detection and other advance features - looking mainly for the three second delay before the boot button changes the eps32 status.

thank you

![PXL_20230801_153026499](https://github.com/darthcloud/BlueRetro/assets/83892045/4402c836-bc5c-46f0-9999-ff670007875a)

**Comment by Colrut** [comment]
HW2 does not seem to work at all with HW1 wiring.

**Reply by darthcloud** [reply]
that's normal, its expect hw2 wiring

**Reply by Colrut** [reply]
Thank you.
